vllm/utils.py

```diff
@@ -25,6 +25,7 @@
 import psutil
 import torch
 import torch.types
+import yaml
```

Comment:
Since this isn't in the standard library, we should add it to `requirements-common.txt`.

Reply:
good catch!